packages/core/test/insured-bridge/e2e/InsuredBridge.e2e.js

```diff
@@ -151,48 +156,107 @@ describe("Insured Bride", () => {
           proposerBondPct,
           identifier
         );
-      await l1BridgePoolFactory.deployTransaction.wait();
+      await l1BridgeAdmin.deployTransaction.wait();
+
+      l1BridgePool = await factory__L1_BridgePool.connect(l1Wallet).deploy(l1BridgeAdmin.address, l1Timer.address);
+      await l1BridgePool.deployTransaction.wait();
 
       l2BridgeDepositBox = await factory__L2_BridgeDepositBox
         .connect(l2Wallet)
-        .deploy(l1BridgePoolFactory.address, minimumBridgingDelay, l2Timer.address, OPTIMISM_GAS_OPTS);
-      // await l2BridgeDepositBox.deployTransaction.wait();
+        .deploy(l1BridgeAdmin.address, minimumBridgingDelay, l2Timer.address, OPTIMISM_GAS_OPTS);
       await l2BridgeDepositBox.deployTransaction.wait();
-    });
-    it("Can whitelist token on L2 from L1 Router", async () => {
-      // Set deposit contract on deposit router.
-      await l1BridgePoolFactory.setDepositContract(l2BridgeDepositBox.address);
 
-      assert.equal(await l1BridgePoolFactory.getDepositContract(), l2BridgeDepositBox.address);
+      // Set deposit contract on deposit Admin.
+      await l1BridgeAdmin.setDepositContract(l2BridgeDepositBox.address);
+    });
+    describe("Cross-domain admin functionality", () => {
+      it("Can whitelist token on L2 from L1", async () => {
+        // Double check the UMA collateral whitelist has the L1 token whitelisted.
+        assert.isTrue(await l1CollateralWhitelist.isOnWhitelist(l1Token.address));
+
+        // Token is not whitelisted before changing via bridge
+        assert.isFalse(await l2BridgeDepositBox.isWhitelistToken(l2Token.address));
+
+        // Wallet should have to tokens on L1 from the mint action and no tokens on L2.
+        const whitelistTx = await l1BridgeAdmin.whitelistToken(
+          l1Token.address, // L1 token.
+          l2Token.address, // Associated L2 token.
+          l1BridgePool.address,
+          10000000 // L2Gas limit. set this to a large number, but less than gas limit, to ensure no L2 revert.
```

Comment:
Should we put this `10000000` into `OPTIMISM_GAS_OPTS`?

Reply:
ideally, yes. can pull it into the constants file.